types/express-handlebars/index.d.ts

```diff
@@ -36,7 +40,7 @@ interface Exphbs {
     getTemplate(filePath: string, options?: PartialTemplateOptions): Promise<Function>;
     getTemplates(dirPath: string, options?: PartialTemplateOptions): Promise<Object>;
     render(filePath: string, context: Object, options?: RenderOptions): Promise<string>;
-    renderView(viewPath: string, optionsOrCallback: any, callback?: () => string): void;
+    renderView(viewPath: string, optionsOrCallback: ExphbsCallback | any, callback?: ExphbsCallback): void;
```

Comment:
There's no point to this change if this is getting union-ed with `any`. Maybe you should create an overload instead.

```ts
renderView(viewPath: string, callback: ExphbsCallback): void;
renderView(viewPath: string, options: any, callback: ExphbsCallback): void;
```

Reply:
And that's why you shouldn't code while tired 😂

Reply:
Do you happen to be a medical professional? I'm constantly tired and I'm hoping to get  a doctor's note out of work 😂